static/src/javascripts/projects/common/modules/commercial/build-page-targeting.js

```diff
@@ -271,7 +270,7 @@ const buildPageTargetting = (
             urlkw: getUrlKeywords(page.pageId),
             rdp: getRdpValue(ccpaState),
             consent_tcfv2: getTcfv2ConsentValue(adConsentState),
-            cmp_interaction: tcfv2EventStatus,
+            cmp_interaction: tcfv2EventStatus || 'na',
```

Comment:
@ioanna0, in the tests `buildPageTargeting()` is called without any arguments. I think a _null_ `tcfv2EventStatus` should translate to the `'na'` string, rather than a _null_ value.